scripts/install.sh

```diff
@@ -187,7 +187,7 @@ err() {
 
 need_cmd() {
     if ! check_cmd "$1"
-    then err "need '$1' (command not found)"
+    then err "Installation halted. Reason: [command not found '$1' - please add this command]"
```

Comment:
```suggestion
    then err "Installation halted. Reason: [command not found '$1' - please install this command]"
```
Not sure if it's better... but I'm more used to read  `install` instead of `add` in that kind of context